README.md

```diff
@@ -55,7 +55,7 @@ interface Logger {
 ### Example
 
 ```js
-const feeds = new Feeds({ serviceId: your_service_id });
+const feeds = new Feeds({ instanceId: your_service_id });
```

Comment:
should probably also change `your_service_id` -> `your_instance_id`. Good catch btw!

Reply:
Ditto! Will fix 🔧